README.md

```diff
@@ -16,5 +16,14 @@ Install using the command line:
 Install-Package Quamotion.TurboJpegWrapper
 ```
 
+#### OS X .NET Core
+Two additional packages are required -
+
+runtime.osx.10.10-x64.Quamotion.TurboJpegWrapper\
+`dotnet add package runtime.osx.10.10-x64.Quamotion.TurboJpegWrapper`
+
+System.Drawing.Common\
+`dotnet add package System.Drawing.Common`
```

Comment:
System.Drawing.Common is already marked as a dependency; see https://www.nuget.org/packages/Quamotion.TurboJpegWrapper/1.0.3-r118. Was it not installed automatically for you?

Reply:
Nope, had to be done again. It's weird as on nuget shows dependency. But in VS it does not. The project targets `Microsoft.NETCore.App (2.0.0)`

<img width="835" alt="screen shot 2018-02-20 at 3 41 58 pm" src="https://user-images.githubusercontent.com/3583020/36418426-d87e74b2-1654-11e8-8442-fb9e97241a46.png">